pkg/cmd/server/admin/create_mastercerts.go

```diff
@@ -113,6 +114,8 @@ func NewCommandCreateMasterCerts(commandName string, fullName string, out io.Wri
 	flags.StringSliceVar(&options.Hostnames, "hostnames", options.Hostnames, "Every hostname or IP that server certs should be valid for (comma-delimited list)")
 	flags.BoolVar(&options.Overwrite, "overwrite", false, "Overwrite all existing cert/key/config files (WARNING: includes signer/CA)")
 
+	flags.IntVar(&options.ExpireDays, "expire-days", options.ExpireDays, "Validity of the certificates in days (defaults to 2 years, you should not change this lightly).")
```

Comment:
`you should not change this lightly`

You can change this value. Even more, making it shorter is perfectly valid. What we should discourage, is to make it longer. So maybe something along "you should not increase this lightly". Or maybe even "WARNING: extending this above default value is highly discouraged".